src/commandargs.cpp

```diff
@@ -5,16 +5,27 @@
 
 static ProgramConfiguration* Config = nullptr;
 static Setting* CurrentState = nullptr;
+String programName;
 
 bool MatchesFlag(String token)
 {
+	if(token.length() < 1)
+	{
+		return false;
```

Comment:
I believe you're using tabs here but the standard is 2 spaces.

Reply:
Whoops, will fix